### PR TITLE
CASMINST-4408: Linting

### PR DESCRIPTION
**Pull request by mharding-hpe**
Ready for review.

* Removed .pyc files which were mistakenly (I assume) committed into the repo
* Usual linting of spelling, formatting, style, grammar, clarity, and consistency issues

No content changes, only linting. A limited number of script files had linting done to them, but it was all fixing spelling errors in comments except one case where a function name had a spelling error. In that case, a global replace of the function name to its correct spelling was done. I searched the rest of the docs-csm repo to make absolutely sure that no other files had a dependency on the function in the script in question.

### CODE REVIEW

File: operations/utility_storage/Remove_Ceph_Node.md [hunk 1]
```diff
@@ -10,78 +10,76 @@ This procedure describes how to remove a Ceph node from the Ceph cluster. Once t
...
-    done
+    ncn-s# for osd in $(ceph osd ls-tree $NODE); do
+              ceph osd down osd.$osd;
+              ceph osd destroy osd.$osd --force;
+              ceph osd purge osd.$osd --force;
+           done
     ```
 
-7. Regenerate Rados-GW Load Balancer Configuration
+1. Regenerate Rados-GW Load Balancer Configuration
```
Comment (nrockershousen) [inline operations/utility_storage/Remove_Ceph_Node.md:70]: This line is missing a period

File: operations/utility_storage/Remove_Ceph_Node.md [hunk 2]
```diff
@@ -10,78 +10,76 @@ This procedure describes how to remove a Ceph node from the Ceph cluster. Once t
...
-3. View the status of each OSD and see where they reside.
+1. View the status of each OSD and see where they reside.
 
     ```bash
-    ceph osd tree
+    ncn# ceph osd tree
     ```
 
-4. Set the `NODE` variable
+1. Set the `NODE` variable
```
Comment (nrockershousen) [inline operations/utility_storage/Remove_Ceph_Node.md:37]: This line is missing a period

File: operations/utility_storage/Remove_Ceph_Node.md [hunk 3]
```diff
@@ -94,69 +92,68 @@ This procedure describes how to remove a Ceph node from the Ceph cluster. Once t
...
 
    On the ***node being removed***
 
     ```bash
-    cephadm rm-cluster --fsid $(cephadm ls|jq -r .[1].fsid) --force
+    ncn-s# cephadm rm-cluster --fsid $(cephadm ls|jq -r .[1].fsid) --force
     ```
 
-10. Remove the node from the crush map
+1. Remove the node from the crush map
```
Comment (nrockershousen) [inline operations/utility_storage/Remove_Ceph_Node.md:151]: - This line is missing a period
- Change "crush" to "CRUSH"

**Review by mtupitsyn — APPROVED** [review]

File: operations/utility_storage/Remove_Ceph_Node.md (hunk 1 above)
Comment (mharding-hpe) [reply operations/utility_storage/Remove_Ceph_Node.md]: Fixed!

File: operations/utility_storage/Remove_Ceph_Node.md (hunk 2 above)
Comment (mharding-hpe) [reply operations/utility_storage/Remove_Ceph_Node.md]: Fixed!

File: operations/utility_storage/Remove_Ceph_Node.md (hunk 3 above)
Comment (mharding-hpe) [reply operations/utility_storage/Remove_Ceph_Node.md]: Both fixed

**Review by gbaker-hpe — APPROVED** [review]

**Pull request merged**